### PR TITLE
PostgreSQL repository and exception translation

### PR DESCRIPTION
**Pull request by dennishendriksen**
Pending work:
- Remove Questionnaire hack from PostgreSQL repository
- Fix reindexing during bootstrapping
- Fix migration during bootstrapping
- Write PostgreSQL repository integration tests
- Enable Transactional annotations in PostgreSQL repository

### CODE REVIEW

File: molgenis-core-ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "MOLGENIS",
-  "version": "1.19.0-SNAPSHOT",
+  "version": "1.20.0-SNAPSHOT",
```
Comment (bartcharbon) [inline molgenis-core-ui/package.json]: shouldn't this be 2.0.0-SNAPSHOT?

File: molgenis-data-postgresql/pom.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.molgenis</groupId>
+		<artifactId>molgenis</artifactId>
+		<version>1.20.0-SNAPSHOT</version>
```
Comment (bartcharbon) [inline molgenis-data-postgresql/pom.xml]: shouldn't this be 2.0.0-SNAPSHOT?

**Comment by bartcharbon** [comment]
the app won't start anymore if no molgenis.version is specified in the properties file, this is the default for "clean install"

**Comment by bartcharbon** [comment]
If I open the permissionmanager I get:

15:25:52.914 [qtp1726759945-139] ERROR o.m.u.a.p.PermissionManagerController - null
java.lang.NullPointerException: null
	at org.molgenis.data.postgresql.PostgreSqlExceptionTranslator.doTranslate(PostgreSqlExceptionTranslator.java:40) ~[classes/:na]

**Comment by bartcharbon** [comment]
I cannot delete single rows from the dataexplorer and not delete data and metadata (data alone works fine)

**Comment by bartcharbon** [comment]
When I upload the "violationunique" I do get the correct error in the importer.
however, the entity does show up in the entity dropdown in the dataexplorer.

(selecting it results in an error: ERROR: relation "violationunique" does not exist)

**Comment by dennishendriksen** [comment]
@bartcharbon I tested this on the meta data edit PoC and this will be solved automagically when the meta data story train has been completed.

**Pull request merged**